consensus/src/dag/tests/dag_network_test.rs

```diff
@@ -126,9 +126,9 @@ async fn test_send_rpc_with_fallback() {
         )
         .await;
 
-    assert_ok!(rpc.next().await.unwrap());
+    assert_ok!(rpc.next().await.unwrap().unwrap().deref());
```

Comment:
what is the deref for?

Reply:
Deref unwraps the DAGRpcResult struct and get the inner Result. I implemented Deref for DAGRpcResult but for some reason I have to explicitly deref here.